cppcache/integration/test/PdxInstanceTest.cpp

```diff
@@ -268,7 +268,7 @@ TEST(PdxInstanceTest, testPdxInstance) {
       pdxTypeFromPdxTypeInstance->equals(*pdxTypeFromPdxTypeInstanceGet, false))
       << "PdxObjects should be equal.";
 
-  EXPECT_EQ(-960665662, pdxTypeInstance->hashcode())
+  EXPECT_EQ(514863279, pdxTypeInstance->hashcode())
```

Comment:
Why did this hash code change? Are we certain it matches the same hash that the server produces for it? If not this can lead to really bad single hop performance.

Reply:
Damn, I did not thought of the fact that it was possible to use a PdxInstance as a key. Really thanks for pointing this out,